libs/sdk-ui-kit/src/Dialog/ConfirmDialogBase.tsx

```diff
@@ -59,58 +60,66 @@ export class ConfirmDialogBase extends DialogBase<IConfirmDialogBaseProps> {
         const headerClassNames = cx("gd-dialog-header", dialogHeaderClassName);
 
         return (
-            <div tabIndex={0} onKeyDown={this.onKeyDown}>
-                <div className={dialogClasses}>
-                    {displayCloseButton ? this.renderCloseButton() : null}
+            <UiFocusTrap autofocusOnOpen={true}>
```

Comment:
Will this auto focus get forwarded to some input in the dialog that has autofocus enabled?

Reply:
so far it works because input comes second so it takes focus on it as last or is really first element in dialog. But FocusTrap itself does not know about autofocusable input and selects really just first focusable element. If it will be an issue, autofocus prop needs to be rewritten from boolean to selector/element or Dialog needs to have its own prop autofocusOnOpen=true/false based on its content